picter-labels.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "name": "priority: critical",
+    "color": "D10069"
+  },
+  {
+    "name": "priority: serious",
+    "color": "D10069"
+  },
+  {
+    "name": "priority: medium",
+    "color": "D10069"
+  },
+  {
+    "name": "priority: low",
+    "color": "D10069"
+  },
+  {
+    "name": "status: blocked",
+    "color": "000000"
+  },
+  {
+    "name": "status: help wanted",
+    "color": "000000",
+    "formerNames": ["help wanted"]
+  },
+  {
+    "name": "status: review needed",
```

Comment:
Not so happy with this one. Either a PR is ready to be reviewed or it is in progress. To manage this manually with a label is sure to be outdated.